sdk/face/azure-ai-vision-face/README.md

```diff
@@ -0,0 +1,429 @@
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-vision-face;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-documentintelligence</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authenticate the client
+
+In order to interact with the Face service, you will need to create an instance of a client class,
+[FaceAsyncClient][face_client_async] or [FaceClient][face_client] by using [FaceClientBuilder][face_client_builder].
+[FaceAdministrationAsyncClient][face_administration_client_async] or [FaceAdministrationClient][face_administration_client] by using [FaceAdministrationClientBuilder][face_administration_client_builder].
+[FaceSessionAsyncClient][face_session_client_async] or [FaceSessionClient][face_session_client] by using [FaceSessionClientBuilder][face_session_client_builder].
+
+An **endpoint** and **credential** are necessary to instantiate the client object.
+
+#### Get the endpoint and API keys
+
+You can find the endpoint and keys for your Face resource using the [Azure Portal][azure_sdk_java_azure_key_credential] or [Azure CLI][get_endpoint_via_azure_cli]:
+
+```bash
+# Get the endpoint for the Face resource
+az cognitiveservices account show --name "resource-name" --resource-group "resource-group-name" --query "properties.endpoint"
+```
+
+Either a regional endpoint or a custom subdomain can be used for authentication. They are formatted as follows:
+
+```
+Regional endpoint: https://<region>.api.cognitive.microsoft.com/
+Custom subdomain: https://<resource-name>.cognitiveservices.azure.com/
+```
+
+A regional endpoint is the same for every resource in a region. A complete list of supported regional endpoints can be consulted [here][regional_endpoints]. Please note that regional endpoints do not support AAD authentication.
+
+A custom subdomain, on the other hand, is a name that is unique to the Face resource. They can only be used by [single-service resources][azure_portal_create_face_account].
+
+```bash
+# Get the API keys for the Face resource
+az cognitiveservices account keys list --name "<resource-name>" --resource-group "<resource-group-name>"
+```
+
+#### Create the client with AzureKeyCredential
+
+To use an API key as the `credential` parameter, pass the key as a string into an instance of [AzureKeyCredential][azure_sdk_java_azure_key_credential].
+
+```java com.azure.ai.vision.face.readme.keyAuthentication
+String endpoint = "https://<my-custom-subdomain>.cognitiveservices.azure.com/";
+String accountKey = "<api_key>";
+FaceClient client = new FaceClientBuilder()
+    .endpoint(endpoint)
+    .credential(new KeyCredential(accountKey))
+    .buildClient();
+```
+
+#### Create the client with an Azure Active Directory credential
```

Comment:
Due to the security enhancement, please make the AAD auth first option.

BTW, I think now it should be called "[Microsoft Entra ID](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id)"

Generally Azure would want to discourage api-key.

Reply:
Got it. Will update it later.

Reply:
I have updated the README.md and make the AAD auth first.

Moreover, I add the following sentences to emphazie the importance of Microsoft Entra ID

`AzureKeyCredential` authentication is used in the examples in this getting started guide because of its' simplicity, but we recommend to authenticate with Microsoft Entra ID using the [azure-identity][azure_sdk_java_default_azure_credential] library. Microsoft Entra ID is more scure and reliable.

Reply:
However, I would like to keep the other sample code to use key authentication as default.
The reason is that
In most secnario, customers who looks for samle code are invegesting solutions for their scenario or protype. More likely, they don't have a concrete idea on what to do. They are seeking some inspiration.
Using Key authentication is much more easier. Using Entra ID is a somewaht difficult. according our experience for customer support and new team member to on board.

I hope that we won't set up a high threshold for our potential customers to try our product.

Although using Entra ID is a trend inside our company, but it's a trend outside now.

Company that regards security as high priority will use Entra ID even without our promote.

I would suggest to keep using Key Authentication for now. If later our company decide to remove all key authentication or discurage our cusotmers to use key authentication.  I will go back to update these samples.

Does it make sense to you?